src/main/java/seedu/address/model/customer/Customer.java

```diff
@@ -63,6 +64,42 @@ public Optional<Address> getAddress() {
         return Optional.ofNullable(address);
     }
 
+    /**
+     * Get total revenue generated from the customer
+     * @return double total revenue
+     */
+    public double getRevenue() {
+        double revenue = 0;
+        for (Commission commission : commissions) {
+            revenue += commission.getFee().fee;
+        }
+        return revenue;
+    }
+
+    /**
+     * Get total number of commission made by this customer
+     * @return total number of commissions
+     */
+    public long getCommissionCount() {
+        return commissions.getSize();
+    }
+
+    /**
+     * Get total number of active commissions made by this customer
+     * @return number of commissions
+     */
+    public long getActiveCommissionCount() {
+        return commissions.getActiveSize();
+    }
+
+    /**
+     * Get last date of customer's commissions
+     * @return
```

Comment:
The @return tag here is empty- is this intentional? I think we should either fill it in or delete the tag.